### PR TITLE
Created separate functions to toggle three components of portfolio

### PR DESCRIPTION
**Pull request by jnnybk**
I cleaned up inconsistent spacing, followed style guide in naming IDs and variables, and created separate functions to display correct information based on the page that user clicks.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -7,30 +7,31 @@
...
+			<table>
+				<th onclick="ToggleReset()">home</th>
+				<th onclick="ToggleAbout()">about me</th>
+				<th onclick="ToggleContact()">contact me</th>
+			</table>
 
-      
-    	<div id="MainPage">
+				
+			<div id="main-page">
```
Comment (mickrobk) [inline portfolio/src/main/webapp/index.html:20]: Nitpick: using a word other than "page" here (and the other similar places) would be more clear, as "page" is overloaded WRT web development. What about "main-section"?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -7,30 +7,31 @@
...
-		<table>
-			<tr>
-				<th onclick=>about me</th>
-			</tr>
-			<tr>
-				<th>contact me</th>
-			</tr>
-		</table>
+			<table>
+				<th onclick="ToggleReset()">home</th>
```
Comment (belraquib) [inline portfolio/src/main/webapp/index.html:14]: can you parameterise these functions?

consider:
Toggle("about-page")
Toggle("contact-page")
Toggle(null)

The first thing toggle can do is hide everything, then if the arg is non-null, look up the arg and show it.

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -12,23 +12,49 @@
...
- */
+
+
+
+function ToggleContact() {
+	const contactElement = document.getElementById("contact-page");
+	const aboutElement = document.getElementById("about-page");
+	const mainElement = document.getElementById("main-page");
+
+	console.log("Function ToggleContact() has been invoked.");
```
Comment (mickrobk) [inline portfolio/src/main/webapp/script.js:23]: Nitpick: Usually it's good to remove debugging logs before commit!

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -13,9 +13,12 @@
...
+#main-page {
+	display: block
 }
-#Contact {
+#contact-page {
   display: none
+}
+#about-page {
+	display: none
 }
```
Comment (belraquib) [inline portfolio/src/main/webapp/style.css:24]: add a newline at the end

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -12,23 +12,29 @@
...
+function setVisibility(currentValue, idName) {
+	if (idName === currentValue) {
+		document.getElementById(currentValue).style.display = "block";
+	} else {
+		document.getElementById(currentValue).style.display = "none";
+	}
+}
+function Toggle(idName) {
+	const sectionList = [ 'contact-section', 'about-section', 'main-section'];
+	for(let section of sectionList) {
```
Comment (belraquib) [inline portfolio/src/main/webapp/script.js:24]: prefer const here. section isn't changing.

**Pull request merged**